functests/performance/updating_profile.go

```diff
@@ -215,6 +241,23 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 	})
 })
 
+func getMcpStatus(conditionType machineconfigv1.MachineConfigPoolConditionType) corev1.ConditionStatus {
```

Comment:
This function looks good for me, how about modifying waitForMcpCondition() for calling this function inside? It will reduce duplicate code.

Reply:
Done.